test/nullable-validation.test.js

```diff
@@ -1,18 +1,17 @@
 'use strict'
 
-const t = require('tap')
-const test = t.test
+const { test } = require('node:test')
 const sget = require('simple-get').concat
 const Fastify = require('..')
 
-test('nullable string', t => {
+test('nullable string', (t, done) => {
   t.plan(3)
   const fastify = Fastify()
   fastify.route({
     method: 'POST',
     url: '/',
     handler: (req, reply) => {
-      t.same(req.body.hello, null)
+      t.assert.deepStrictEqual(req.body.hello, null)
```

Comment:
Since we are testing the hello value in the object i think we can use `strictEqual`
```suggestion
      t.assert.strictEqual(req.body.hello, null)
```